src/Attachment/index.ts

```diff
@@ -9,6 +9,7 @@
 
 /// <reference path="../../adonis-typings/index.ts" />
 
+import { join } from 'node:path'
```

Comment:
Hi @RomainLanz . Thank you for the PR.

`node:path` is not available in Node 14 and I still want to support it. Please can you use `path` instead?